utils/s3/client.cc

```diff
@@ -89,6 +90,19 @@ future<> ignore_reply(const http::reply& rep, input_stream<char>&& in_) {
 client::client(std::string host, endpoint_config_ptr cfg, semaphore& mem, global_factory gf, private_tag, std::unique_ptr<aws::retry_strategy> rs)
         : _host(std::move(host))
         , _cfg(std::move(cfg))
+        , _creds_invalidation_timer([this] {
+            s3l.info("Invalidating creds");
```

Comment:
Not very informative. Better to have a longer message saying smth like "Failed to update credentials in the background, dropping outdated credentials so that requests will try to re-obtain them synchronously later" or alike